pages/profiles/[publicKey]/nfts.tsx

```diff
@@ -260,6 +271,13 @@ export const NFTGrid = ({
           marketplace={marketplace}
         />
       ))}
+      {hasMore && (
+        <div>
```

Comment:
Is the wrapper `div` helpful here?

Reply:
yep it needs a DOM element to latch on to